src/App.css

```diff
@@ -1,38 +1,69 @@
 .App {
-  text-align: center;
+  display: grid;
+  grid-template-rows: min-content 1fr;
+  gap: var(--spacing--gap);
+  padding: var(--spacing--gap);
+  width: 100vw;
+  height: 100vh;
 }
 
-.App-logo {
-  height: 3em;
-  padding: .5em;
-}
+.App-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
 
-.React-logo {
-  height: 50vh;
-  width: 50vw;
+  padding: var(--spacing--gap);
```

Comment:
There ends up being kind of a lot of empty space at the top of the page. Now that I look more closely, this is in keeping with the mockups you rendered in #5; I forgot I had been looking at scaled images! Maybe the goal is to have a counterbalance against the density that's coming later? I only noticed because my previous screenshots fit in 440x440px, and with this change that's no longer enough vertical space!

Reply:
All that extra space comes from the amount of invisible padding in the spinning react image:

<img width="513" alt="image" src="https://user-images.githubusercontent.com/4731/224431087-3691ae80-53e0-489e-ba2e-cb54e61ffd4b.png">

Being a temporary stand-in anyway, I'm going to not worry about it. We can create a version of the logo without the padding if we want!

Reply:
Ah sorry, I meant the space above the PDC logo / home button! With these changes the PDC logo is larger and has more padding above it:

before:
![image](https://user-images.githubusercontent.com/1494855/224431968-fde59f31-04c1-4fd5-818c-0c9a7f88a51b.png)

after:
![image](https://user-images.githubusercontent.com/1494855/224431899-e506afec-51da-49cf-8d16-329dde82bb67.png)

Reply:
Oh, yes! It is quite a bit roomier than before, but in keeping with the mockup.

We can adjust this down if we'd like at smaller viewports, though we're intentionally unconcerned with the mobile experience. I do see how this would be nicer for screenshotting, though.